database/src/main/kotlin/no/nav/su/se/bakover/database/person/PersonPostgresRepo.kt

```diff
@@ -108,12 +108,13 @@ internal class PersonPostgresRepo(
         return dataSource.withSession { session ->
             """
                SELECT
-                    sak.fnr søkersFnr,
-                    behandling.behandlingsinformasjon->'ektefelle'->>'fnr' epsFnr
-               FROM revurdering
-               INNER JOIN behandling on behandling.id = revurdering.behandlingid
-               INNER JOIN sak ON sak.id = behandling.sakId
-               WHERE revurdering.id=:revurderingId
+                    s.fnr søkersFnr,
+                    v.behandlingsinformasjon->'ektefelle'->>'fnr' epsFnr
+               FROM revurdering r
+               INNER JOIN behandling_vedtak bv on bv.vedtakId = r.opprinneligVedtakId
```

Comment:
Ref. kommentaren min om opprinneligVedtakId. Hvis `opprinneligVedtakId` alltid vil peke på den originale søknadsbehandlingen vil dette fungere :)

Reply:
For en revurdering av en revurdering vil `opprinneligVedtakId` peke til vedtaket som kom av den forrige revurderingen; ikke til den aller første søknadsbehandlingen. Mener du da at dette ikke vil funke?

Reply:
Nei, jeg tror jeg må ha tenkt feil. Jeg tror dette blir rett!